descriptions.md

```diff
@@ -1056,6 +1056,8 @@ A generic interface for event registration and delivery used in a number of the
 || `Event` ||| The named event to listen for. |
 || `Data` ||| The event listener. |
 | off() ||| RTE5 | Deregisters all registrations, for all events and listeners. |
+| off(Event) ||| RTE5 | Removes all registrations that match the specified event. |
```

Comment:
> Removes all registrations

The other methods use "deregister" rather than "remove", so for consistency I would go with "deregisters".

Reply:
Fixed